test/e2e/user_management_test.go

```diff
@@ -99,3 +194,27 @@ func (s *userManagementTestSuite) TestUserDisabled() {
 		verifyResourcesProvisionedForSignup(s.T(), s.awaitility, userSignup, revisions, "basic")
 	})
 }
+
+func (s *userManagementTestSuite) createUserSignupAndAssertAutoApproval(specApproved bool) (*v1alpha1.UserSignup, *v1alpha1.MasterUserRecord) {
+	id := uuid.NewV4().String()
+	return s.createAndCheckUserSignup(specApproved, "testuser"+id, "testuser"+id+"@test.com", approvedAutomatically()...)
+}
+
+func newBannedUser(host *wait.HostAwaitility, email string) *v1alpha1.BannedUser {
```

Comment:
could you please move this function to `base_user_test.go` file? it's called only there

Reply:
Sure, however I should also point out that there's an equivalent `newUserSignup` function in `usersignup_test`.

Reply:
but `newUserSignup` is used inside of `usersignup_test`, so it's fine having it there

anyway, thanks for moving the `newBannedUser` function